BlackStone8960/Musician-First
# Added the flow to create Stripe Connected Account and Payment flow, fixed problem related to Redux

Enabled producers to create their accounts in Stripe and created payment flow using API.
Also, I fixed some problems related with Redux that will be happened when editing user profile because it's necessary to implement these features.

If you have time, please test these flow. 
I'm setting it as testing environment, so you won't pay real money. Don't worry.

 - **Please use this URL https://musicianfirstcanada-staging.herokuapp.com/ to test payment functionalities** 

 - To create account, you can start it by selecting producer as the occupation when signing up new account, and follow the steps (You don't have to input real information in this flow)
 
 - To test payment, 
1. Choose one producer who has Stripe account (for testing, you can choose your own account where you created Stripe account) 
2. Push "Hire {producer name}" button 
3. Select one of three hiring plans 
4. Input testing card information referencing this page https://stripe.com/docs/testing (e.g. 4242-4242-4242-4242)
5. If it's succeeded, you should proceed to a page that shows Payment successful

Also I edited Redux code related to profile editing, so you can test editing correctly work or not as well if you have time.

Thank you!
